cpp/include/cudf/table/table_view.hpp

```diff
@@ -32,7 +33,7 @@
  * passed by value.
  */
 
-namespace cudf {
+namespace CUDF_EXPORT cudf {
```

Comment:
For my own education, in what situations do we need to set the visibility for the namespace instead of individual functions?

Reply:
By placing on the namespace it captures all definitions inside the braces. This allows us to mark all the functions as public without having to modify each definition.